ts/call-action-form.tsx

```diff
@@ -104,9 +104,11 @@ export class CallActionForm extends React.Component<Props, State> {
 			} as State);
 			this.props.setModal("loading");
 			var campaign = this.getCampaign(this.props.campaignId);
+			var zipField = (document.getElementsByName('input_zip').item(0) as HTMLInputElement);
```

Comment:
I would defer to @sirpengi on proper form here (although he's on vacation currently), but I think doing a cross-component DOM lookup is something that should likely be avoided, because `PetitionForm` won't necessarily even exist on the same page, like when navigating to `?call=1`.